### PR TITLE
Include the trademark, and fix some css in the footer.

### LINKED ISSUE #250: Trademark registration notice
The Jupyter Trademark has now been issued and we need to provide a notice of registration on our website. Our lawyer has recommended the following notation:

"®" or "Reg. U.S. Pat. & TM Off." or "Registered in U.S. Patent & Trademark Office."

This can probably go in the site footer.

### PR DESCRIPTION
**Pull request by Carreau**
Closes #250

### CODE REVIEW

**Comment by Carreau** [comment]
Also add a livereload command for developement.

File: _includes/footer.html [hunk 1]
```diff
@@ -17,7 +17,9 @@
                     {% endfor %}
                     <!--{{ site.time}}-->
                 </ul>
-                <p>Copyright © {{site.time | date: '%Y' }}  Project Jupyter - Last updated {{site.time | date: "%a, %b %d, %Y" }}</p>
+                <p>Copyright © {{site.time | date: '%Y' }}  Project Jupyter –
+                Last updated {{site.time | date: "%a, %b %d, %Y" }}</p>
+                <p>The Jupyter Trademark is registered in U.S. Patent &amp; Trademark Office.</p>
```
Comment (ellisonbg) [inline _includes/footer.html]: is registered with the U.S.

File: _includes/footer.html [hunk 2]
```diff
@@ -17,7 +17,9 @@
                     {% endfor %}
                     <!--{{ site.time}}-->
                 </ul>
-                <p>Copyright © {{site.time | date: '%Y' }}  Project Jupyter - Last updated {{site.time | date: "%a, %b %d, %Y" }}</p>
+                <p>Copyright © {{site.time | date: '%Y' }}  Project Jupyter –
+                Last updated {{site.time | date: "%a, %b %d, %Y" }}</p>
+                <p>The Jupyter Trademark is registered with U.S. Patent &amp; Trademark Office.</p>
```
Comment (ellisonbg) [inline _includes/footer.html]: "with" -> "with the" ;-)

Comment (Carreau) [reply _includes/footer.html]: Dang, I'll get this right ! One day !

**Comment by ellisonbg** [comment]
Got it! Thanks!

**Comment by Carreau** [comment]
Thanks !

**Pull request merged**